src/relay/transforms/dynamic_to_static.cc

```diff
@@ -139,6 +139,24 @@ class DynamicToStaticMutator : public MixedModeMutator {
            }
            return Expr(nullptr);
          }},
+        {Op::Get("dyn.strided_slice"),
+         [](const CallNode* call_node) {
+           if (const ConstantNode* begin = call_node->args[1].as<ConstantNode>()) {
+             if (const ConstantNode* end = call_node->args[2].as<ConstantNode>()) {
```

Comment:
It would be cleaner to pull these definitions out of the if statements, and then check whether they are null or not in one if statement, though potentially slower